app/com/arpnetworking/kairos/client/models/MetricsQueryResponse.java

```diff
@@ -0,0 +1,309 @@
+/**
+ * Copyright 2017 Smartsheet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.kairos.client.models;
+
+import com.arpnetworking.commons.builder.OvalBuilder;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import net.sf.oval.constraint.Min;
+import net.sf.oval.constraint.NotNull;
+import org.joda.time.DateTime;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Model class to represent a metrics query response from KairosDB.
+ *
+ * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+ */
+public final class MetricsQueryResponse {
+    @JsonAnyGetter
+    public Map<String, Object> getOther() {
+        return _other;
+    }
+
+    public List<Query> getQueries() {
+        return _queries;
+    }
+
+    private MetricsQueryResponse(final Builder builder) {
+        _other = builder._other;
+        _queries = Lists.newArrayList(builder._queries);
+    }
+
+    private final Map<String, Object> _other;
+    private final List<Query> _queries;
+
+    /**
+     * Implementation of the builder pattern for {@link MetricsQueryResponse}.
+     *
+     * @author Brandon Arp (brandon dot arp at smartsheet dot com)
+     */
+    public static final class Builder extends OvalBuilder<MetricsQueryResponse> {
+        /**
+         * Public constructor.
+         */
+        public Builder() {
+            super(MetricsQueryResponse::new);
+        }
+
+        /**
+         * Adds an "unknown" parameter. Optional.
```

Comment:
This is the second time I've seen this description; really don't get it. The parameter isn't "unknown", you specify the key.

Reply:
This is typically called by the JsonAnySetter.  So, we don't really know it now.  It happens at runtime during deserialization.